File: LeetCode/168/main.py
"""
DIFFICULTY : easy
TAGS : math, string
"""


class Solution:
    def convertToTitle(self, n: int) -> str:
        ans = []
        while n:
            n, r = divmod(n - 1, 26)
            ans.append(chr(r + ord("A")))
        return "".join(reversed(ans))

File: LeetCode/168/main.cpp
class Solution {
public:
  string convertToTitle(int n) {
    int rem = 0;
    string ans = "";

    while (n > 26) {
      n--;
      rem = n % 26;
      ans = char('A' + rem) + ans;
      n /= 26;
    }

    n--;
    ans = char('A' + n) + ans;

    return ans;
  }
};